main.tf

```diff
@@ -42,4 +42,63 @@ resource "aws_iam_user_policy" "ssm_policy" {
 
   user   = aws_iam_user.user.id
   policy = data.aws_iam_policy_document.ssm_parameter_doc[count.index].json
+  name   = format("terraform_read_ssm_parameters_%d", count.index + 1)
+}
+
+# IAM policy documents that allow the EC2 actions needed for packer to create
+# AMIs.  This will be applied to the IAM user we are creating.
+data "aws_iam_policy_document" "ec2_packer_doc" {
+  count = var.add_packer_permissions ? 1 : 0
+
+  statement {
+    effect = "Allow"
+
+    actions = [
```

Comment:
I believe that if these are not in a required order they should be done alphabetically to improve readability.

Reply:
Good catch- thanks! Fixed in https://github.com/cisagov/aws-parameter-store-read-only-iam-user-tf-module/pull/2/commits/6b63539cde23fc35a5a01a6646b78d150d449e2d